app/scripts/modules/core/utils/SystemTimezone.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import * as moment from 'moment';
+
+import { SETTINGS } from 'core/config/settings';
+
+export class SystemTimezone extends React.Component<any, any> {
+  public render() {
+    const zone = SETTINGS.defaultTimeZone || 'America/Los_Angeles';
```

Comment:
we do this `'SETTINGS.defaultTimeZone || 'America/Los_Angeles';` in a few different places in the code. Can't we just set the default value in `SETTINGS` itself?